client/app/components/packages/deis/attached-documents.hbs

```diff
@@ -0,0 +1,17 @@
+<@form.Section @title="Attached Documents">
+  <p>
+   To complete this submission, attach the complete draft 
+   Environmental Impact Statement (DEIS) and Final Scope of Work. 
+   Upload all chapters and appendices as individual files. 
+   The maximum size for a document is 50MB
+  </p>
+
+  <Packages::Attachments
+    @package={{@model}}
+    @fileManager={{@model.fileManager}}
+    data-test-section="attachments"
+  />
+</@form.Section>
+
+
+
```

Comment:
tiny nit but why all this extra space? should always only be 1 carriage return after the last non-empty character.

Reply:
Not sure. Is there a good way to check/valid these in VS code while coding?